### PR TITLE
Issue 3027 - Fetch container views endpoint 

### LINKED ISSUE #3027: Endpoints to fetch views from containers
### Description
endpoint to fetch view list from federations was adding in #2789

We need to do the same for containers.

NOTE: this issue will branch from #2789 for simplicity and to avoid conflicts

### Goals
- [ ] Fetching view list from containers should succeed

### Tasks
- [ ] Create endpoint to fetch views list
- [ ] refactor file-type to utils

### PR DESCRIPTION
**Pull request by ChristosTsiotsias**
This fixes #3027

#### Description
Fetch container views endpoint implemented.
Swagger documentation and unit - e2e tests added.

#### Test cases
The user should be able to fetch container views provided they pass correct details
The user should get an empty array if there are no views in a container
The user should get an error if they pass invalid details

### CODE REVIEW

File: backend/src/v5/middleware/dataConverter/multer.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const imageFilter = (req, file, cb) => {
 const ensureFileIsImage = async (req, res, next) => {
 	const fileBuffer = req.file?.buffer;
 	if (fileBuffer) {
-		const type = await FileType.fromBuffer(fileBuffer);
+		const type = await fileTypeFromBuffer(fileBuffer);
```
Comment (carmenfan) [inline backend/src/v5/middleware/dataConverter/multer.js:64]: Can we extrapolate this a bit more

maybe split it into 2 functions

one that returns the extension, the other returns the MIME type

**Review by carmenfan — CHANGES_REQUESTED** [review]
Looks fine, just a small change on the file-type refactoring

File: backend/src/v5/middleware/dataConverter/multer.js (hunk 1 above)
Comment (ChristosTsiotsias) [reply backend/src/v5/middleware/dataConverter/multer.js:64]: something like this ?
image

Comment (carmenfan) [reply backend/src/v5/middleware/dataConverter/multer.js:64]: sure, i don't think you need the top one though

You just need another function that returns you the mime type

**Pull request merged**